pkg/importer/util.go

```diff
@@ -42,3 +44,23 @@ func StreamDataToFile(dataReader io.Reader, filePath string) error {
 	}
 	return nil
 }
+
+// UnArchiveTar unarchives a tar file and streams its files
+// using the specified io.Reader to the specified destination.
+func UnArchiveTar(reader io.Reader, destDir string) error {
```

Comment:
Maybe we can re-use the untar from clonertarget.go. Move it into util or something, then importing from repo can use it as well.

Reply:
It's already in util.
If you think I should change anything else, please say so.

Reply:
Oops didn't realize that, well I guess it would nice if you can update the clonertarget to use this one instead of the one it uses. Then at the same time you can steal the unit test I have for that untar method for this one.

Reply:
I guess that it would be more right to do it on a separate PR.